app/scicrunch_process_results.py

```diff
@@ -1,7 +1,7 @@
 import importlib
 import json
 import re
-
+from flask import jsonify
```

Comment:
Why make this change?

Reply:
The jsonify() function in flask returns a flask.Response() object that already has the appropriate content-type header 'application/json' for use with json responses. Whereas, the json.dumps() method will just return an encoded string, which would require manually adding the MIME type header.

https://stackoverflow.com/a/13172658/2217801

Reply:
A good enough reason.